cmd/helm/dependency_build.go

```diff
@@ -67,6 +68,7 @@ func newDependencyBuildCmd(out io.Writer) *cobra.Command {
 	f := cmd.Flags()
 	f.BoolVar(&dbc.verify, "verify", false, "verify the packages against signatures")
 	f.StringVar(&dbc.keyring, "keyring", defaultKeyring(), "keyring containing public keys")
+	f.BoolVar(&dbc.recursive, "recursive", false, "recursive mode")
```

Comment:
I've put a bit more descriptive text here now, but I could not expand the word "recursive" too much 😄 .